app/models/form/lettings/subsections/property_information.rb

```diff
@@ -3,7 +3,7 @@ def initialize(id, hsh, section)
     super
     @id = "property_information"
     @label = "Property information"
-    @depends_on = [{ "non_location_setup_questions_completed?" => true }]
+    @depends_on = [{ "non_location_setup_questions_completed?" => true, "supported_housing_and_renewal?" => false }]
```

Comment:
Hm, so this depends_on is used to determine if the section should be unlocked or not. Is this the correct place to put `supported_housing_and_renewal?` condition?
I'd imagine we don't want to display it at all/don't want to route to any of the pages in this section as opposed to just lock it

Reply:
I'm not sure I follow. Do you mean having the displayed_in_tasklist? method below should be enough?

Reply:
Also relevantly I went through all the pages in the section and they all depend on being general needs or not renewal, although perhaps that could change in future?

Reply:
if this has a smell for you then perhaps the "proper" solution would be to make sure postcode_known is set to nil, did you see that comment? and do you know if setting it to nil definitely would be desired behaviour once it's no longer routed to?

Reply:
Yeah I'm not sure in terms of setting the postcode known. I'm also not a 100% sure why it being false impacts anything. 
Also, what does this line meant to achieve? Sorry, I'm just confused

Reply:
https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/1336/files

linking this here for discussion. It appeared to me to be a similar situation, where we don't wish to display an entire subsection, dependent on these conditions